oh-my-bash.sh

```diff
@@ -144,6 +144,14 @@ unset -v _omb_init_files _omb_init_file
 # Load the theme
 if [[ $OSH_THEME == random ]]; then
   _omb_util_glob_expand _omb_init_files '"$OSH"/themes/*/*.theme.sh'
+
+  # Remove ignored themes from the list
+  for _omb_ignore_theme in random "${OMB_THEME_RANDOM_IGNORED[@]}"; do
+    _omb_ignore_path="$OSH/themes/$_omb_ignore_theme/$_omb_ignore_theme.theme.sh"
+    _omb_init_files=("${_omb_init_files[@]/$_omb_ignore_path}")
```

Comment:
This leaves an empty element, which can be selected and attempted to be sourced to cause an error with `source ""`. You should remove the element explicitly.